src/main/java/in/projecteka/consentmanager/clients/model/PatientLinkReferenceResult.java

```diff
@@ -21,7 +22,7 @@
 @Data
 public class PatientLinkReferenceResult {
     private UUID requestId;
-    private String timestamp;
+    private LocalDateTime timestamp;
```

Comment:
We need to update the timestamp format to LocalDateTime in the HIP before. Otherwise, an error will be thrown in the consent_manager on serialize/deserialize.